validator/src/validation/range.rs

```diff
@@ -2,32 +2,45 @@ use crate::validation::Validator;
 
 /// Validates that a number is in the given range
 ///
-/// TODO: see if can be generic over the number type
+#[deprecated(since = "0.12.0", note = "Please use the validate_range_generic function instead")]
```

Comment:
same, just change the code and we will bump the version

Reply:
> and we will bump the version

I could also change the `length` validator to take a path as well before we push the next version. There is not issue for it but that should be simple enough and useful as well :) (But probably to much for this PR)

Reply:
Go for it in that PR if you want

Reply:
I will :)